packages/e2eTesting/integration/checkbox.spec.ts

```diff
@@ -0,0 +1,75 @@
+/// <reference types="cypress" />
```

Comment:
Normally typescript should be properly setup so that we don't have to add this annotation to each test file we create.
```suggestion
```

Reply:
Got it! will remove it from here and all the other tests.